### PR TITLE
Bug fixes 

### LINKED ISSUE #43: `heatmap_mutation_frequency_bin` lost rotation of the column title
The `heatmap_mutation_frequency_bin` used to plot the column titles with rotation 90. But this feature was lost at some point. See this parameter:
https://github.com/morinlab/GAMBLR.viz/blob/6563146016f1231d39356ad4d4ecbbd7185207f9/R/heatmap_mutation_frequency_bin.R#L347

The default setting in the earlier versions of this function were set to 90, so the gene names do not overlap when single regions are plotted. Probably, this was changed intentionally because it was making some version of the resulting plot better. So why don't we convert this to a user-configurable argument that can be dynamically set based on user's needs. The default can be 0 to respect current behaviour.

### PR DESCRIPTION
**Pull request by Kdreval**
Fixes #43 
Also fixes the issue #42 

# Pull Request Checklists

**Important:** When opening a pull request, keep only the applicable checklist and delete all other sections.

### Required

- [X] I ensured not to prepend the functions existing in both GAMBLR.data and GAMBLR.results with `<package>::function()` syntax.
- [X] I ensured not to add GAMBLR.data or GAMBLR.results to the `@import` section of the documentation

### CODE REVIEW

File: R/heatmap_mutation_frequency_bin.R [hunk 1]
```diff
@@ -33,6 +33,7 @@
 #' @param cluster_cols_heatmap Boolean.  Default FALSE.
 #' @param show_gene_colours Boolean. Whether to add heatmap annotation colours for each region. Default FALSE.
 #' @param label_regions_by Specify which feature of the regions to label the heatmap with. Heatmap will be split according to this value, and ordered by factor levels if the specified column is a factor. Default name.
+#' @param label_regions_rotate Specify degree by which the label in the previous parameter will be rotated. Default 0 (no rotation).
```
Comment (vladimirsouza) [inline R/heatmap_mutation_frequency_bin.R:36]: Need to include that only 0, 90 and 270 values are possible ([ComplexHeatmap::Heatmap](https://www.rdocumentation.org/packages/ComplexHeatmap/versions/1.10.2/topics/Heatmap)).

**Comment by vladimirsouza** [comment]
Since this PR is fixing prettyOncoplot, can you fix `splitColumnName` parameter documentation? This parameter has no default value. 
```
#' @param splitColumnName Optional argument to indicate which metadata column to split on. Default is set to pathology.
```

**Comment by Kdreval** [comment]
Thanks @vladimirsouza , both of the comments are resolved in the latest commit. This is ready for re-review

**Review by vladimirsouza — APPROVED** [review]
Thanks!!

**Pull request merged**